test/pom.xml

```diff
@@ -46,5 +46,15 @@
             <artifactId>junit-jupiter-api</artifactId>
             <version>${junit.jupiter.version}</version>
         </dependency>
+        <dependency>
```

Comment:
you don't need to change this file

Reply:
This happened automatically I think. I just added to the pom.xml in rest-utils-test

Reply:
this pom file isn't related to core/pom.xml